lib/expr.c

```diff
@@ -3895,6 +3920,7 @@ parse_near_options(efs_info *q,
 {
   const char *end;
   data->options.near.max_interval = grn_atoi(start, q->str_end, &end);
+  data->options.near.min_interval = DEFAULT_MIN_INTERVAL;
   if (start == end) {
```

Comment:
Please don't split `grn_atoi()` and `start == end` check.
They are related.

Reply:
Oh, sorry. I didn't noticed.

Fixed.